pipelines/release/tarball.groovy

```diff
@@ -709,7 +709,7 @@ def String buildScript(
     ciDir
   ) +
   util.dedent("""
-    curl -sSL ${util.newinstallUrl()} | bash -s -- -cb
```

Comment:
Is the implication that there is some sort of newinstallUrl function we could have changed instead? Or at least delete it to avoid confusion elsewhere.

Reply:
Yes there is, but I didn't want to change it as it might have had other consequences.  It turns out that the only other place it is used is in the build-newinstall pipeline, which also has problems, so perhaps I will change it there and get rid of the function (which is no longer necessary as the lsstinstall URL is constant for all time).

Reply:
Nope, that requires changes to a separate Dockerfile, so I'd rather not fold it into this ticket.